### ISSUE TITLE
No context switch on switching between versions

### ISSUE THREAD

**Issue by ShahAnuj2610** [appbaseio/Docs#31]
I am on some components doc (React v3), now if I switch to React v2, then the doc context shouldn't change

**Comment by jyash97** [comment]
True, it's a good enhancement but it may lead to a 404 page if we redirect to a component which has been removed in version v2 or v3 which isn't good so we will need to find a proper way to figure this out.
